app/src/main/res/values/strings.xml

```diff
@@ -68,6 +69,99 @@
         <item>CH2</item>
     </string-array>
 
+    <string-array name="multimeter_params">
+        <item>CH1</item>
+        <item>CH2</item>
+        <item>CH3</item>
+        <item>MIC</item>
+        <item>CAP</item>
+        <item>AN8</item>
+        <item>RES</item>
+        <item>C</item>
```

Comment:
@CloudyPadmal can you help this? what is a suitable word?

Reply:
@cweitat  this is to be put into the config file. User won't see this names. I have added more comprehensible names as titles in another string - array